core/drivers/scmi-msg/platform_weaks.S

```diff
@@ -0,0 +1,56 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
```

Comment:
Comment to the name of this file. Arm32 assembly files normally are named with a `_a32.S` suffix.

Reply:
Ok, I'll create a `_a32.S` file and a `_a64.S` file.